pkg/owner/manager.go

```diff
@@ -352,31 +352,35 @@ func getOwnerInfo(ctx, logCtx context.Context, etcdCli *clientv3.Client, ownerPa
 	}
 	if err != nil {
 		logutil.Logger(logCtx).Warn("etcd-cli get owner info failed", zap.Error(err))
-		return "", nil, op, 0, errors.Trace(err)
+		return "", nil, op, 0, 0, errors.Trace(err)
 	}
 	if len(resp.Kvs) == 0 {
-		return "", nil, op, 0, concurrency.ErrElectionNoLeader
+		return "", nil, op, 0, 0, concurrency.ErrElectionNoLeader
 	}
 
 	var ownerID []byte
 	ownerID, op = splitOwnerValues(resp.Kvs[0].Value)
 	logutil.Logger(logCtx).Info("get owner", zap.ByteString("owner key", resp.Kvs[0].Key),
 		zap.ByteString("ownerID", ownerID), zap.Stringer("op", op))
-	return string(resp.Kvs[0].Key), ownerID, op, resp.Kvs[0].ModRevision, nil
+	return string(resp.Kvs[0].Key), ownerID, op, resp.Kvs[0].CreateRevision, resp.Kvs[0].ModRevision, nil
 }
 
-// GetOwnerKey gets the owner key information.
-func GetOwnerKey(ctx, logCtx context.Context, etcdCli *clientv3.Client, etcdKey, id string) (string, error) {
-	ownerKey, ownerID, _, _, err := getOwnerInfo(ctx, logCtx, etcdCli, etcdKey)
+// GetOwnerKeyWithCreateRevision gets the owner key and createRevision information.
+func GetOwnerKeyWithCreateRevision(
```

Comment:
```suggestion
func GetOwnerKey(
```
WithXXX mostly means we are passing XXX to the function

Reply:
> WithXXX mostly means we are passing XXX to the function

modified.